### DISCUSSION TITLE
Roadmap of sharing actions privately

### DISCUSSION THREAD

**Discussion by andyfeller** [community/community#25575] [Actions]
<p>According to the <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-actions#about-actions" rel="nofollow noopener">documentation,</a> “You can write your own actions to use in your workflow or share the actions you build with the GitHub community. To share actions you’ve built, your repository must be public.”</p>
<p>This will not work for my organization due to legal and security policies, so I hope there is news about supporting private sharing of actions at least within an organization.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p> Hi,</p>
<p>Many thanks for your feedback!  </p>
<p>There is a same <a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.community%2Ft5%2FGitHub-Actions%2FGithub-action-action-in-private-repository%2Ftd-p%2F29722&amp;data=02%7C01%7Cv-wadzho%40microsoft.com%7C2de81c0a9e4549fe67e908d76cba7b73%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637097420502853997&amp;sdata=euZ%2BKjY1XTL6TJR1%2Bk%2FXnZ5Vhc1uU3dQTOods14jq1I%3D&amp;reserved=0" rel="nofollow noopener">ticket</a> for tracking, it’s recommended to vote on it. Any update/roadmap will be informed once there is a response for the feature request, thanks.</p>
